### ISSUE TITLE
ValueError: could not convert string to float

### ISSUE THREAD

**Issue by vidu3k333** [hako-mikan/sd-webui-lora-block-weight#114]
```
Traceback (most recent call last):
  File "E:\AI art\webui\modules\extra_networks.py", line 135, in activate
    extra_network.activate(p, extra_network_args)
  File "E:\AI art\webui\extensions-builtin\Lora\extra_networks_lora.py", line 33, in activate
    unet_multiplier = float(params.positional[2]) if len(params.positional) > 2 else te_multiplier
ValueError: could not convert string to float: 'OUTD'
```

**Comment by hako-mikan** [comment]
https://github.com/hako-mikan/sd-webui-lora-block-weight/issues/73
https://github.com/hako-mikan/sd-webui-lora-block-weight/issues/78
https://github.com/hako-mikan/sd-webui-lora-block-weight/issues/93
https://github.com/hako-mikan/sd-webui-lora-block-weight/issues/98
https://github.com/hako-mikan/sd-webui-lora-block-weight/issues/107
https://github.com/hako-mikan/sd-webui-lora-block-weight/issues/109
https://github.com/hako-mikan/sd-webui-lora-block-weight#updates%E6%9B%B4%E6%96%B0%E6%83%85%E5%A0%B1

**Comment by ddoscv** [comment]
Hi, so if I want to use "OUTD", it would be something like <lora:"lora name":1:OUTD> or <lyco:"lora name":1:1:lbw=OUTD>?
But I noticed that even my lycoris files displays as <lora:"lora name":1>
I also noticed that even using the same prompts and settings and seed, I'm not getting the same results anymore after I updated from SD1.2 to SD1.6. In fact, the image is much worse even I followed this format like you mentioned:
<lora:"lora name":1:IN02>  
<lyco:"lora name":1:1:lbw=IN02> 

Any idea of what's could be the problem is?
And sorry this might seem similar to other posts here already.
